packages/@react-aria/interactions/src/usePress.ts

```diff
@@ -301,15 +301,21 @@ export function usePress(props: PressHookProps): PressResult {
           return;
         }
 
+        // iOS safari fires pointer events from VoiceOver with incorrect coordinates/target.
+        // Ignore and let the onClick handler take care of it instead.
+        // https://bugs.webkit.org/show_bug.cgi?id=222627
+        // https://bugs.webkit.org/show_bug.cgi?id=223202
+        if (isVirtualPointerEvent(e.nativeEvent)) {
+          return;
+        }
```

Comment:
Interesting, note to self to try ComboBox tray w/ iOS voiceover since I think something similar was happening there as well